### ISSUE TITLE
error: Unrenderable variable

### ISSUE THREAD

**Issue by restagner** [Orange-OpenSource/hurl#3514]
I'm attempting to capture a response using `jsonpath` and then provide that response to the next endpoint as payload.  However, when I attempt to run the script I get an error

```
variable <statement_request_ids> with value [173088,173089,173090,0,0,0,0] can not be rendered
```

Here is my script
```
POST http://hostname:8666/statementgentool

[
    payload here
 ]
 

HTTP 200
[Captures]
statement_request_ids: jsonpath "$[:].StatementRequestId"



POST http://hostname:8666/statementgentool/query

{{statement_request_ids}}


HTTP 200

```

What am I doing wrong?

**Comment by jcamiel** [comment]
Hi @restagner 

Unfortunately, arrays and object variables can (still) not be rendered as it, just number, strings, boolean (nodes of arrays and object)
Depending on how is structured your response, if you know the size of your arrays, you can do something like this

```hurl
POST http://hostname:8666/statementgentool
HTTP 200
[Captures]
id0: jsonpath "$[:].StatementRequestId[0]"
# this works too
# id0: jsonpath "$[:].StatementRequestId" nth 0 
id1: jsonpath "$[:].StatementRequestId[1]"
id2: jsonpath "$[:].StatementRequestId[2]"
# etc...

POST http://hostname:8666/statementgentool/query
[{{id0}}, {{id1}}, ... {{idn}}]
```

**Comment by restagner** [comment]
Hi @jcamiel ,

Thanks for the feedback.  I will use your suggestion.  Thankfully, the list I am dealing with is fixed.  

Any idea/thoughts if arrays will ever be supported in Hurl?

**Comment by jcamiel** [comment]
Yes arrays / object are already supported, what's we have not designed is how to render an array.

You may want to render a list like `a|b|c` and not as a JSON array `["a","b","c"]`.

We want to implement a way to render a list as you want, in in the (long) TODO list!

**Comment by kevalin** [comment]
Hi @jcamiel ,
I'm having the same confusion and I'm wondering when hurl will be able to support the rerender object/array variable returned from the previous request.
thanks.

**Comment by kevalin** [comment]
> Hi [@jcamiel](https://github.com/jcamiel) , I'm having the same confusion and I'm wondering when hurl will be able to support the rerender object/array variable returned from the previous request. thanks.

If this TODO hasn't started yet, can I contribute to this feature and provide a PR?

**Comment by jcamiel** [comment]
Hi @kevalin 
Unfortunately, no ETA for this issue right now. It's not opened to contribution (see [CONTRIBUTING.md#how-can-you-help](https://github.com/Orange-OpenSource/hurl/blob/master/CONTRIBUTING.md#how-can-you-help-)), you can look for ["open-to-contributing" labeled issues](https://github.com/Orange-OpenSource/hurl/issues?q=is%3Aissue%20state%3Aopen%20label%3Aopen-to-contribution) if you want to help
